src/Source.jl

```diff
@@ -159,7 +160,7 @@ function Source(dsn::DSN, query::AbstractString; weakrefstrings::Bool=true, noqu
     ODBC.API.SQLSetStmtAttr(stmt, ODBC.API.SQL_ATTR_ROWS_FETCHED_PTR, rowsfetched, ODBC.API.SQL_NTS)
     types = [ODBC.API.SQL2C[ctypes[x]] for x = 1:cols]
     source = ODBC.Source(schema, dsn, query, columns, 100, rowsfetched, 0, boundcols, indcols, csizes, types, Type[longtexts[x] ? ODBC.API.Long{T} : T for (x, T) in enumerate(juliatypes)], supportsreset == 1)
-    rows != 0 && fetch!(source)
+    (isa(rows, Missing) || rows != 0) && fetch!(source)
```

Comment:
Should be `(ismissing(rows) || rows != 0) ...`. Just curious: is there a problem with setting `rows = 0` instead of `missing`?

Reply:
@omus Tested it, doesn't seem to be any problem with using 0 instead of missing if we wanted to.

Reply:
Ok, we should set it to `0` then. Worth making a comment at the assignment.